deepset_cloud_sdk/_service/files_service.py

```diff
@@ -276,20 +280,21 @@ def _validate_file_paths(file_paths: List[Path]) -> None:
         :raises ValueError: If the file paths are invalid.
         """
         logger.info("Validating file paths and metadata.")
-        allowed_suffixes = {".txt", ".json", ".pdf"}
+        allowed_suffixes = {".csv", ".docx", ".html", ".json", ".md", ".txt", ".pdf", ".pptx", ".xlsx", ".xml"}
         for file_path in file_paths:
             if file_path.suffix.lower() not in allowed_suffixes:
                 raise ValueError(
-                    f"Invalid file extension: {file_path.suffix}. You can upload TXT and PDF files. Metadata files should have the `.meta.json` extension."
-                )
-            if file_path.suffix.lower() == ".json" and not str(file_path).endswith(".meta.json"):
-                raise ValueError(
-                    f"JSON files are only supported for metadata files. Make sure you follow this naming format for your metadata files: '<file_name>.meta.json'. Got {file_path.name}."
+                    f"Invalid file extension: {file_path.suffix}. Supported file types: csv, docx, html, json, md, txt, pdf, pptx, xlsx, xml. "
```

Comment:
nit: how about using an f-string and then pass in the list of support types as string? That would avoid an outdating of the exception message